deploy/olm-catalog/3scale-operator/3scale-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: 3scale-operator.v0.3.0
+  namespace: placeholder
+  annotations:
+    repository: https://github.com/3scale/3scale-operator
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    description: 3scale Operator to provision 3scale and publish/manage API
+    containerImage: quay.io/3scale/3scale-operator:v0.3.0
+    createdAt: 2019-05-30T22:40:00Z
+    support: Red Hat, Inc.
+    tectonic-visibility: ocs
+    alm-examples: >-
+       [{"apiVersion":"apps.3scale.net/v1alpha1","kind":"APIManager","metadata":{"name":"example-apimanager"},"spec":{"wildcardDomain":"example.com"}},
+       {"apiVersion":"apps.3scale.net/v1alpha1","kind":"APIManager","metadata":{"name":"example-apimanager"},"spec":{"apicast":{"image":"quay.io/3scale/apicast:nightly"},"backend":{"image":"quay.io/3scale/apisonator:nightly"},"system":{"image":"quay.io/3scale/porta:nightly"},"wildcardDomain":"example.com","zync":{"image":"quay.io/3scale/zync:nightly"}}},
```

Comment:
I'd remove the nightly example setup from the examples.
Users should not use that setup because is inherently unstable and for tests. We'll provide a YAML with that content and should really only be used by our development teams to perform development or some occasional unsupported test from external people